templates/master/00-master/ovirt/files/ovirt-haproxy.yaml

```diff
@@ -11,7 +11,7 @@ contents:
       creationTimestamp:
       deletionGracePeriodSeconds: 65
       labels:
-        app: openstack-infra-api-lb
+        app: kni-infra-api-lb
```

Comment:
Seems like we're missing an `ovirt` namespace in our list of namespaces?

Reply:
we need to consult with kni folks about this... maybe for some reason they prefer to use  "openshift-kni-infra" namespace for ovirt, but imho it's better to create a separate namespace

Reply:
I don't really mind which namespace is used. having it named after oVirt makes sense to me

Reply:
Let's put ovirt